packages/api/amplify_api_dart/lib/src/graphql/web_socket/blocs/is_windows/is_windows_io.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:io';
+
+import 'package:aws_common/aws_common.dart';
+
+/// Is the environment Windows vm (not Web on Windows).
+bool isWindows() {
+  return !zIsWeb && Platform.isWindows;
```

Comment:
Probably could drop `zIsWeb`, although I guess it doesn't hurt.

Reply:
yeah, went back and forth on it as a readability thing but I guess will just drop it